devices/generic/items/state_enhanced_hue_item.json

```diff
@@ -0,0 +1,15 @@
+{
+  "schema": "resourceitem1.schema.json",
+  "id": "state/enhanced_hue",
+  "datatype": "UInt16",
+  "access": "R",
+  "public": false,
+  "description": "The current color hue.",
+  "parse": {
+    "fn": "zcl",
+    "ep": 0,
+    "cl": "0x0300",
+    "at": "0x4000",
+    "eval": "R.item('state/hue).val = Attr.val"
```

Comment:
This throws an JS exception since `'state/hue`  is a unterminated string.
Note all JS code is test compiled during load and errors are reported via `--dbg-ddf=1`.

```
DDF failed to compile JS: /home/mpi/src/deCONZ_V2_XX/src/plugins/de_web/devices/dresden_elektronik/hive.json
SyntaxError: unterminated string (line 1, end of input)
```

The output is not too detailed but gives a hint.

To be honest I would propose not to add new `state/enhanced_hue` and `state/enhanced_colormode` items. The existing `state/hue` already reflects enhanced hue when available. For `state/colormode` we often map enhanced color mode values as well. Since we can redefine parsing directly in the DDF, if needed, we don't need extra items here imho.